draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -195,7 +187,7 @@ Included in the model are representations of configuration, states, and telemetr
 
 Drawing from existing IETF models and potentially complementing that with input from other standard or industrial forum models (ITU-T, OpenConfig , ONF TAPI etc.) , this model offers enhanced uniform structuring and naming.
 
-This draft introduces the concept of "Coherent Pluggable Manifest", where it represents the capabilities of pluggable, maintained in a repository. This document also covers gap analysis of current IETF drafts and other SDOs on coherent Pluggable attributes and provides the complete lifecycle of a coherent pluggable from operators' approval through viability assessment to deployment.
+This draft introduces the concept of "Coherent Pluggable Repository", where it represents the capabilities of pluggable, maintained in a repository. This document also covers gap analysis of current IETF drafts and other SDOs on coherent Pluggable attributes and provides the complete lifecycle of a coherent pluggable from operators' approval through viability assessment to deployment.
 
 This document also covers an analysis of the gap between current IETF drafts and the corresponding works in other standards bodies and industry. The lifecycle of a coherent pluggable from operators' approval, viability assessment to deployment and monitoring is also covered.
 
```

Comment:
I can't find O-PNC and P-PNC being defined in RFC845. Besides, the term "The control functions specializing in management/control of optical and photonic functions (virtual or physical)." is quite nebulous. E.g.:
- is interface up/down a photonic function because the laser is switched off?
- are uncorrected errors packet or optical "specialities"?
- is a transponder with a non coherent interface on client side and a coherent one on the media side supposed to be managed/controlled by the P-PNC?